HTML/test3.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="myApp">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Angular Example</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.8.2/angular.min.js"></script>
+    <script>
+        var app = angular.module('myApp', []);
+        app.controller('myCtrl', function($scope) {
+            $scope.message = "Hello, Angular!";
+            $scope.changeMessage = function() {
+                $scope.message = "You clicked the button!";
+                $scope.message = "You clicked the button!";
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2208262726>#007008</a><span> - 07/04/2024, 07:11 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Duplicate assignment to '$scope.message' in the 'changeMessage' function.
</div>
<div id="fix">
<b>Fix:</b> Remove the duplicate assignment to '$scope.message' in the 'changeMessage' function.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -12,7 +12,6 @@
              $scope.message = "You clicked the button!";
```
</div>
</details>
</div>
<br>


</div>